src/components/general/Tag.tsx

```diff
@@ -0,0 +1,21 @@
+import {PropsWithChildren} from "react";
+import {cn} from "@/lib/utils";
+
+interface TagProps extends PropsWithChildren {
+  bgColor?: string;
+  textColor?: string;
+}
+
+export const Tag = ({children, ...props}: TagProps) => {
+  const hexRegex = /^#[a-fA-F\d]{6}$/
```

Comment:
fine for now, but when we get a lot of regexes, we should place these in one const file with explanations for what they do